cloud_edition/flexibility_mode/docs/system_administration.rst

```diff
@@ -1,72 +1,59 @@
+| Alias                      | Argument                 | Action                                                                                                                                           |
++============================+==========================+==================================================================================================================================================+
+| ``partners_mysql``         | ``status|start|restart`` | Show status or start/restart mysql daemon                                                                                                        |
++----------------------------+--------------------------+--------------------------------------------------------------------------------------------------------------------------------------------------+
+| ``partners_elasticsearch`` | ``status|start|restart`` | Show status or start/restart elasticsearch daemon                                                                                                |
++----------------------------+--------------------------+--------------------------------------------------------------------------------------------------------------------------------------------------+
+| ``partners_php7.3-fpm``    | ``status|start|restart`` | Show status or start/restart php-fpm daemon (Command name can vary depending on php version)                                                     |
++----------------------------+--------------------------+--------------------------------------------------------------------------------------------------------------------------------------------------+
+| ``partners_clear_cache``   |                          | Clear PIM cache properly. Stops php-fpm and job consumers, deletes PIM cache folder, warms up PIM cache and restarts php-fpm and job consumers   |
++----------------------------+--------------------------+--------------------------------------------------------------------------------------------------------------------------------------------------+
+| ``partners_systemctl``     | ``status|start|restart`` | Show status or start/restart job consumers, see below for more details.                                                                          |
++----------------------------+--------------------------+--------------------------------------------------------------------------------------------------------------------------------------------------+
 
+Third-pary software installation
+--------------------------------
 
-Please note that if **no configuration** file exist for a given daemon identifier,
-the daemon will consider it has to consume **any** elements in the queue. This is the default
-behavior.
+You can install any software you need as long as they are standalone do not require the use of apt.
```

Comment:
I doubt that giving them our blessing to install anything (as long as they are capable of doing it) is a good idea.

I suggest we say here we don't install anything on demand for security and for maintenance reason. Which technically doesn't prevent them from installing something by themselves, under their sole responsibility. 

As for installing PHP packages with apt: we have a variable to do it, so we could offer it here. But I think it's a feature we don't want neither to advertise nor to maintain.